src/sentry/dynamic_sampling/latest_release_booster.py

```diff
@@ -20,6 +23,62 @@ class TooManyBoostedReleasesException(Exception):
     pass
 
 
+@dataclass(frozen=True)
+class CachedBoostedRelease:
+    id: int
+    timestamp: float
+    environment: Optional[str]
+
+    def to_augmented(self, release: Release, project_id: int) -> "AugmentedBoostedRelease":
+        return AugmentedBoostedRelease(
+            id=self.id,
+            timestamp=self.timestamp,
+            environment=self.environment,
+            version=release.version,
+            platform=Platform(self._get_project_platform_from_release(release, project_id)),
+        )
+
+    @staticmethod
+    def _get_project_platform_from_release(release: Release, project_id: int) -> Optional[str]:
+        try:
+            return release.projects.get(id=project_id).platform  # type:ignore
+        except Project.DoesNotExist:
+            # If we don't find the project of this release we just default to having no platform name in the
+            # BoostedRelease.
+            return None
+
+
+@dataclass(frozen=True)
+class AugmentedBoostedRelease(CachedBoostedRelease):
```

Comment:
```suggestion
class BoostedRelease(CachedBoostedRelease):
```
why not just call it `BoostedRelease` and `BasicBoostedRelease` or `SimpleBoostedRelease`.
May be we can use it not only in redis cache context in future.

Reply:
This is why is called augmented, it means that it contains more data than the basic one. Semantically it would make more sense to have `BoostedRelease` and then `AugmentedBoostedRelease` as child.

Reply:
this word confuse me a lot, may be try `Extended`?

Reply:
> augmented - make (something) greater by adding to it; increase.
"he augmented his summer income by painting houses"
>having been made greater in size or value.
"augmented pensions for those retiring at 65"

From english dict :)

Reply:
In the english dictionary -> having been made greater in size or value

Reply:
Lets do extended then